### ISSUE TITLE
<InfoAdditions>in application xml for macOS

### ISSUE THREAD

**Issue by atmofunk** [Gamua/Adobe-Runtime-Support#674]
Currently, `<InfoAdditions>` is exclusive to the `<iPhone`> tag in our application xml files.   

However, macOS *also* uses an info.plist file, and as of right now I cant seem to include any additions to that .plist without opening the packaged app after building, editing the file, and being forced to re-code-sign the application for distribution.   

My specific use case here is adding an application protocol for launching the app from a url.

Its a clunky/janky workflow as it exists today and it would be great to have the option to affect the contents of info.plist during the build phase!

**Comment by ajwfrost** [comment]
Yes this is getting very much required... we'll look at how we can incorporate this sort of thing into ADT, to make this all a little easier..

**Comment by ajwfrost** [comment]
Added in 33.1.1.406, just uploaded to the website.
Thanks for the suggestion!

**Comment by atmofunk** [comment]
I finally upgraded to .406 to try this new feature (ty!), and it looks like it still doesn't work?  I tried

`<application>
    ...
    <macOS>
        <InfoAdditions>
            <![CDATA[  my stuff ]]>
         </InfoAdditions>
    </macOS>
    ...
</application> `

...and it responds with _error 103: application.macOS is an unexpected element/attribute_ and exits with error 12

I also tried `<macos</macos>` for case sensitivity -- no dice.

Am i putting it in the wrong place?
